src/extensions/KeyboardNavigationExtension/KeyboardNavigationExtension.ts

```diff
@@ -62,7 +62,7 @@ export class KeyboardNavigationExtension extends Module {
 
       // Seeks back x seconds.
       case KeyCodes.LEFT_ARROW:
-        let prevTime = this.getState().currentTime - SKIP_CURRENTTIME_OFFSET;
+        let prevTime = this.getState().currentTime - ((this.instance.config.ui.rewind > 0) ? this.instance.config.ui.rewind : SKIP_CURRENTTIME_OFFSET);
```

Comment:
Let's define this in the default config, then we don't need to check the value here. See https://github.com/matvp91/indigo-player/blob/master/src/createConfig.ts

I think rewindOffset and forwardOffset would make more sense here as variable name.

Would you mind having a seperate PR for the skip forward & backward feature?